src/myuw-search.html

```diff
@@ -28,30 +28,41 @@
         display: flex;
         flex: auto;
         height: 40px;
-        padding: 0 8px;
+        padding: 0 8px 0 12px;
         font-size: 1rem;
         overflow: hidden;
         border: none;
-        border-top-left-radius: 5px;
-        border-bottom-left-radius: 5px;
+        border-top-left-radius: 3px;
+        border-bottom-left-radius: 3px;
         margin: 0;
     }
 
     #submit {
-        border-top-right-radius: 5px;
-        border-bottom-right-radius: 5px;
+        padding: 0;
+        position: relative;
+        border-top-right-radius: 3px;
+        border-bottom-right-radius: 3px;
         border: none;
         border-left: 1px solid;
-        border-color: rgba(0,0,0,0.5);
+        border-color: rgba(0,0,0,0.15);
         color: #333;
         background: #fff;
         width: 56px;
+        height: 40px;
```

Comment:
what does the inclusion of a height give us here? I didn't really notice a difference

Reply:
It's not strictly necessary but I noticed that the search box had its height explicitly set to 40px and since this PR adds some more position styles to the button and its icon an explicit height would make that math easier to understand